commands/Transfering/transfer.js

```diff
@@ -36,8 +36,32 @@ function successMessage(res, amount){
   broadcastOperation(amount, user, transferred_to);
 }
 
+function canRemoveByAngryPoints(removalPoints) {
+  if (removalPoints > 0) {
+    return false
+  }
+  var angryPointsLimit = Libs.ResourcesLib.userRes("angryPointsLimit")
```

Comment:
angryPointsLimit - I think it still not good name

angryPointsMaxLimit - is more good